game/txt/hlp/pennfunc.hlp

```diff
@@ -4580,7 +4580,8 @@ See also: json()
   For <type>...   <data> should be...
   null            not given
   boolean         one arg, either "true", "1", "false" or "0"
-  string          one arg, any string, including an empty string
+  string          one arg, any string, including an empty string - downgrades to ansi markup
```

Comment:
This isn't very clear - 'converts markup to ANSI escape codes' or similar would be better I think. (What does it do with Pueblo?)

Reply:
It does whatever render(...,ansi) does.

Reply:
Which removes pueblo.

Reply:
New description added. Is this clearer?